src/commands/ban.ts

```diff
@@ -3,6 +3,7 @@ import { parseUserMention } from '../utils/parse';
 
 export default {
     name: 'ban',
+    level: 'admin',
```

Comment:
Adding on from my previous comment, I feel like that in this case, the `adminOnly` boolean can be used instead of `level: admin`.